curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/660ae3eeef9ad289bece426b.md

```diff
@@ -0,0 +1,640 @@
+---
+id: 660ae3eeef9ad289bece426b
+title: Step 11
+challengeType: 0
+dashedName: step-11
+---
+
+# --description--
+
+In the previous projects, you used regular functions. But in the rest of the projects, you will be working with <dfn>arrow functions</dfn>. The next few steps will focus on teaching you the basics of arrow functions.
+
+An arrow function is an anonymous function expression and a shorter way to write functions. Anonymous means that the function does not have a name. Arrow functions are always anonymous.
+
+Here is the basic syntax:
+
+```js
+() => {}
+```
+
+To create a named arrow function, you can assign the function to a variable:
+
+```js
+const exampleFunction = () => {
+  // code goes here
+}
+```
+
+Create a new named arrow function called `printGreeting`. Inside the body of that function use the `console.log()` method to print the string `"Hello there!"`.
+
+# --hints--
+
+You should have a function called `printGreeting`.
+
+```js
+assert.isFunction(printGreeting);
+```
+
+You should use arrow syntax to create the `printGreeting` function.
+
+```js
+assert.match(code, /(let|const)\s+printGreeting\s*=\s*\(.*\)\s*=>\s*{\s*[\s\S]*}\s*;?/)
+```
+
+Your `printGreeting` function should include a `console.log('Hello there!)`.
+
+```js
+assert.match(printGreeting.toString(), /console\.log\(['"]Hello\s+there!['"]\)/);
```

Comment:
Can we also allow backticks as that's also valid? I as a learner can go crazy and decided to test that out.

```suggestion
assert.match(printGreeting.toString(), /console\.log\(['"`]Hello\s+there!['"`]\)/);
```